Codegen/test/resources/input/TypeCobol/ProcedureCall-Public.rdz.tcbl

```diff
@@ -45,15 +47,21 @@
       * __________________________________________________
       * OK : proper parameter list (TCRFUN_CALL_PRIVATE_ONLY_SAME_PGM)
            CALL Pgm1PrivateValidateDateFormat
-                    INPUT      somedate someformat
+                    INPUT      somedate
+                               someformat
+                               length of somedate
+                               length of YYYY of somedate
```

Comment:
It seems that you have replaced somedate::YYYY by YYYY of somedate. The use of the qualified name separator "::" is authorized in this context. So it means there is an issue with code generation with LENGTH OF